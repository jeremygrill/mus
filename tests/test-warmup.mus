Int main()
{
	Int i;
	Note g;	
	Note f;
	Note e;
	Note d;
	Note c;
	Seq s;

	s = $([(60,60|4)]);
	i = 0;

	while (i < 12)
	{
		g = (67 + i,60|1);
		f = (65 + i,60|1);
		e = (64 + i,60|1);
		d = (62 + i,60|1);
		c = (60 + i,60|1);

		s = s + [(66 + i,20|1)];
		s = s + [(65 + i,30|1)];
		s = s + [(64 + i,40|1)];
		s = s + [(62 + i,50|1)];
		s = s + [c, e, g];
		s = s + [c, e, g];
		s = s + [c, f, g];
		s = s + [c, f, g];
		s = s + [(60 + i,60|2),(64 + i,60|2),(67 + i,60|2)];
		s = s + [(61 + i,80|2),(65 + i,80|2),(68 + i,80|2)];

		i = ++i;
	}

	prints(s);
	plays(s);
	return 0;
}