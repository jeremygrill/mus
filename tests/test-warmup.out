$([(60,60|4)][(66,20|1)][(65,30|1)][(64,40|1)][(62,50|1)][(67,60|1)(64,60|1)(60,60|1)][(67,60|1)(64,60|1)(60,60|1)][(67,60|1)(65,60|1)(60,60|1)][(67,60|1)(65,60|1)(60,60|1)][(67,60|2)(64,60|2)(60,60|2)][(68,80|2)(65,80|2)(61,80|2)][(67,20|1)][(66,30|1)][(65,40|1)][(63,50|1)][(68,60|1)(65,60|1)(61,60|1)][(68,60|1)(65,60|1)(61,60|1)][(68,60|1)(66,60|1)(61,60|1)][(68,60|1)(66,60|1)(61,60|1)][(68,60|2)(65,60|2)(61,60|2)][(69,80|2)(66,80|2)(62,80|2)][(68,20|1)][(67,30|1)][(66,40|1)][(64,50|1)][(69,60|1)(66,60|1)(62,60|1)][(69,60|1)(66,60|1)(62,60|1)][(69,60|1)(67,60|1)(62,60|1)][(69,60|1)(67,60|1)(62,60|1)][(69,60|2)(66,60|2)(62,60|2)][(70,80|2)(67,80|2)(63,80|2)][(69,20|1)][(68,30|1)][(67,40|1)][(65,50|1)][(70,60|1)(67,60|1)(63,60|1)][(70,60|1)(67,60|1)(63,60|1)][(70,60|1)(68,60|1)(63,60|1)][(70,60|1)(68,60|1)(63,60|1)][(70,60|2)(67,60|2)(63,60|2)][(71,80|2)(68,80|2)(64,80|2)][(70,20|1)][(69,30|1)][(68,40|1)][(66,50|1)][(71,60|1)(68,60|1)(64,60|1)][(71,60|1)(68,60|1)(64,60|1)][(71,60|1)(69,60|1)(64,60|1)][(71,60|1)(69,60|1)(64,60|1)][(71,60|2)(68,60|2)(64,60|2)][(72,80|2)(69,80|2)(65,80|2)][(71,20|1)][(70,30|1)][(69,40|1)][(67,50|1)][(72,60|1)(69,60|1)(65,60|1)][(72,60|1)(69,60|1)(65,60|1)][(72,60|1)(70,60|1)(65,60|1)][(72,60|1)(70,60|1)(65,60|1)][(72,60|2)(69,60|2)(65,60|2)][(73,80|2)(70,80|2)(66,80|2)][(72,20|1)][(71,30|1)][(70,40|1)][(68,50|1)][(73,60|1)(70,60|1)(66,60|1)][(73,60|1)(70,60|1)(66,60|1)][(73,60|1)(71,60|1)(66,60|1)][(73,60|1)(71,60|1)(66,60|1)][(73,60|2)(70,60|2)(66,60|2)][(74,80|2)(71,80|2)(67,80|2)][(73,20|1)][(72,30|1)][(71,40|1)][(69,50|1)][(74,60|1)(71,60|1)(67,60|1)][(74,60|1)(71,60|1)(67,60|1)][(74,60|1)(72,60|1)(67,60|1)][(74,60|1)(72,60|1)(67,60|1)][(74,60|2)(71,60|2)(67,60|2)][(75,80|2)(72,80|2)(68,80|2)][(74,20|1)][(73,30|1)][(72,40|1)][(70,50|1)][(75,60|1)(72,60|1)(68,60|1)][(75,60|1)(72,60|1)(68,60|1)][(75,60|1)(73,60|1)(68,60|1)][(75,60|1)(73,60|1)(68,60|1)][(75,60|2)(72,60|2)(68,60|2)][(76,80|2)(73,80|2)(69,80|2)][(75,20|1)][(74,30|1)][(73,40|1)][(71,50|1)][(76,60|1)(73,60|1)(69,60|1)][(76,60|1)(73,60|1)(69,60|1)][(76,60|1)(74,60|1)(69,60|1)][(76,60|1)(74,60|1)(69,60|1)][(76,60|2)(73,60|2)(69,60|2)][(77,80|2)(74,80|2)(70,80|2)][(76,20|1)][(75,30|1)][(74,40|1)][(72,50|1)][(77,60|1)(74,60|1)(70,60|1)][(77,60|1)(74,60|1)(70,60|1)][(77,60|1)(75,60|1)(70,60|1)][(77,60|1)(75,60|1)(70,60|1)][(77,60|2)(74,60|2)(70,60|2)][(78,80|2)(75,80|2)(71,80|2)][(77,20|1)][(76,30|1)][(75,40|1)][(73,50|1)][(78,60|1)(75,60|1)(71,60|1)][(78,60|1)(75,60|1)(71,60|1)][(78,60|1)(76,60|1)(71,60|1)][(78,60|1)(76,60|1)(71,60|1)][(78,60|2)(75,60|2)(71,60|2)][(79,80|2)(76,80|2)(72,80|2)])
